scripts/notebook_launcher.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+import papermill as pm
+import argparse
+from azureml.core import Run
+
+os.system(sys.executable + ' -m ipykernel install --user --name ipyazureml')
+
+def execute_notebook(notebook_path, parameters):
+    pm.execute_notebook(
+        notebook_path,
+        'outputs/output.ipynb',
+        kernel_name='ipyazureml',
```

Comment:
question here, is it mandatory to install the kernel pyazureml? will this run with the default kernel: https://github.com/Microsoft/Recommenders/blob/master/tests/notebooks_common.py#L8?

Reply:
based on tests @loomlike and I worked on - it will work on the 'python3' kernel as long as ipykernel is there to support the default kernel.